### DISCUSSION TITLE
Freetrack output not available

### DISCUSSION THREAD

**Discussion by lolmuele** [opentrack/opentrack#1448] [Q&A]
I have recently built opentrack on linux , but the only output options are udp, flightgear and joystick.
is this normal ? if yes how do i get the freetrack option since neither joystick nor flightgear work well with dcs.
help appreciated!

**Comment by sthalik** [comment]
Wine output is the Linux equivalent. You need to set the CMake cache variable.

**Comment by itspngu** [comment]
For those wondering what "setting the CMake cache variable" means, it's explained [on the very bottom of the "Building on Linux" wiki page](https://github.com/opentrack/opentrack/wiki/Building-on-Linux#compiling-and-running-opentrack), in the Manjaro section (why there and not in a distro-independent section that's less likely overlooked?)

![image](https://user-images.githubusercontent.com/34752364/163038098-a22ca7fc-6788-4857-98c5-6956cced6a7e.png)

**Reply by nana964** [reply]
Hello I tried all this solution but still not work ( I still haven't wine to propose ) I'm in Arch linux. Thanks
